wallet/addresses.go

```diff
@@ -791,6 +806,11 @@ func (w *Wallet) newChangeAddress(ctx context.Context, op errors.Op, persist per
 // policy.
 func (w *Wallet) NewChangeAddress(ctx context.Context, account uint32) (stdaddr.Address, error) {
 	const op errors.Op = "wallet.NewChangeAddress"
+
+	// Imported voting accounts must not be used for change.
+	if err := w.notVotingAcct(ctx, op, account); err != nil {
```

Comment:
Maybe this should've been added to `newChangeAddress` instead? It seems there's a call path via grpc that can in fact cause the voting account to be used in a change address, by using the voting account number as change destination in a call to the rpcserver's `ConstructTransaction`: internally, that calls `Wallet.NewUnsignedTransaction` which cashes out to using the unexported `newChangeAddress`.

Reply:
Ok, will move it back to the unexported method.

Reply:
Moved.